classes/patreon_frontend.php

```diff
@@ -44,27 +38,13 @@ function __construct() {
 	
 	}
 
-	public function showPatreonButton() {
-
-		global $wp;
-
-		$client_id = get_option('patreon-client-id', false);
-
-		$login_with_patreon = get_option('patreon-enable-login-with-patreon', false);
-		$admins_editors_login_with_patreon = get_option('patreon-enable-allow-admins-login-with-patreon', false);
-
-		if($client_id == false) {
-			return '';
-		}
-
-		$href = self::patreonMakeLoginLink($client_id);
-		
-		echo '<div class="patreon-login-refresh-button">'.self::patreonMakeLoginButton().'</div>';
-	
-	}
 	function patreonEnqueueJs() {
-		wp_register_script( 'patreon-wordpress-js', PATREON_PLUGIN_ASSETS.'/js/app.js', array( 'jquery' ) );
-		wp_enqueue_script( 'patreon-wordpress-js', PATREON_PLUGIN_ASSETS.'/js/app.js', false );
+		// wp_register_script( 'patreon-wordpress-js', PATREON_PLUGIN_ASSETS.'/js/app.js', array( 'jquery' ) );
```

Comment:
Do we still need these?

Reply:
Currently, in near term, no. We can remove them. We may end up adding them back in the coming months depending on what comes up.